core/arch/arm/include/mm/core_memprot.h

```diff
@@ -106,4 +106,9 @@ void *phys_to_virt_io(paddr_t pa);
  */
 paddr_t virt_to_phys(void *va);
 
+/*
+ * Return runtime usable address
```

Comment:
"Return runtime usable address, irrespective of whether the MMU is enabled or not"